docs/extend/plugins_authorization.md

```diff
@@ -58,7 +58,7 @@ respectively.
 
 ## Default user authorization mechanism
 
-If TLS is enabled in the [Docker daemon](../security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
+If TLS is enabled in the [Docker daemon](https://docs.docker.com/engine/security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
```

Comment:
That should point to the URL, including the file extension. Markdown files are different because they are the "source" to generate the HTML pages